be/src/olap/rowset/segment_v2/page_builder.h

```diff
@@ -75,9 +70,8 @@ class PageBuilder {
 
     // This api is for release the resource owned by builder
     // It means it will transfer the ownership of some resource to other.
-    // This api is always called after finish
-    // and should be followed by reset() before reuse the builder
-    virtual void release() = 0;
+    // This api should be followed by reset() before reuse the builder
+    virtual OwnedSlice release() = 0;
```

Comment:
`finish()` is a better name and you should move the method declaration right after `add()` and before `reset()`to be consistent with the order of invocation.